timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/PaxosUseCase.java

```diff
@@ -35,15 +34,6 @@ public Client resolveClient(Client client) {
         }
     },
 
-    // <data-directory>/leaderPaxos/multiLeaderPaxos/<client>/{acceptor/learner}
-    LEADER_FOR_EACH_CLIENT(
-            MULTI_LEADER_PAXOS_NAMESPACE, Paths.get(LEADER_PAXOS_NAMESPACE, MULTI_LEADER_PAXOS_NAMESPACE)) {
-        @Override
-        public Client resolveClient(Client client) {
-            return client;
-        }
-    },
-
```

Comment:
Maybe spicy? Start-up checks should only check the top level folder exists or not